solution/ui/regulations/eregs-vite/src/components/navigation/NavBtn.vue

```diff
@@ -46,54 +46,3 @@ export default {
     },
 };
 </script>
-
-<style lang="scss" scoped>
-.direction-btn {
-    border: none;
```

Comment:
These deleted Vue components were vestigial -- unused and left over from the old prototype site.